README.md

```diff
@@ -1,5 +1,5 @@
 # Dynamic - CLI
->  A CLI utility that can connect to an external databases.
+>  A CLI utility that can connect to external databases.
```

Comment:
Either it should be :  
1. to an external database
2. to external databases.
![image](https://user-images.githubusercontent.com/43691873/90099378-ec070e00-dd57-11ea-8372-c7fc9031c251.png)